src/components/color/index.js

```diff
@@ -57,6 +58,23 @@ color.combine = function(front, back) {
     return tinycolor(fcflat).toRgbString();
 };
 
+// find a color interpolating between two endpoints,
+// in rgba space.
+color.lightColor = function(front, back, backFraction) {
```

Comment:
Since we already have `tinycolor` as a dependency, why not use `tinycolor.mix`?

https://github.com/bgrins/TinyColor#color-utilities